### PR TITLE
Feature/lcov master

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Install lcov master branch version for gcc9

### CODE REVIEW

File: Dockerfile.centos [hunk 1]
```diff
@@ -61,3 +65,8 @@ RUN wget -qO- https://github.com/ccache/ccache/releases/download/v3.7.7/ccache-3
   && make -j \
   && make install \
   && cd ../ && rm -rf ccache-*
+
+# Install lcov to /usr/local/bin
+# require master to make gcc9 happy
+RUN git clone https://github.com/linux-test-project/lcov.git \
```
Comment (yixinglu) [inline Dockerfile.centos]: limit branch and depth

File: Dockerfile.centos [hunk 2]
```diff
@@ -27,13 +26,18 @@ RUN yum update -y && yum install -y centos-release-scl epel-release \
     wget \
     xz \
     zlib-devel \
+# For lcov
```
Comment (yixinglu) [inline Dockerfile.centos]: add spaces for indent

File: Dockerfile.centos (hunk 1 above)
Comment (Shylock-Hg) [reply Dockerfile.centos]: Can't limit depth which lead a strange error.

Comment (Shylock-Hg) [reply Dockerfile.centos]: Can't limit the depth which lead strange error that make vesoft/nebula-dev:centos7 not workable.

File: Dockerfile.centos [hunk 3]
```diff
@@ -61,3 +65,8 @@ RUN wget -qO- https://github.com/ccache/ccache/releases/download/v3.7.7/ccache-3
   && make -j \
   && make install \
   && cd ../ && rm -rf ccache-*
+
+# Install lcov to /usr/local/bin
+# require master to make gcc9 happy
+RUN git clone --depth=1 --branch=master https://github.com/linux-test-project/lcov.git \
```
Comment (yixinglu) [inline Dockerfile.centos]: I think we should download `lcov` sources with specified version from their [releases](https://github.com/linux-test-project/lcov/releases) page. Only in this way, the compiled binary of lcov in docker image is same as before.

Comment (Shylock-Hg) [reply Dockerfile.centos]: The latest release version is not workable for gcc9.

Comment (Shylock-Hg) [reply Dockerfile.centos]: If so , I can specify the commit.

**Pull request merged**